Defines global max and default value for max-runs (#892)
tektoncd/pipelines-as-code: docs/content/docs/install/settings.md

  This will provide us to give extra IPS (ie: 127.0.0.1) or networks (127.0.0.0/16)
  separated by commas.

* `max-keep-run-upper-limit`

>   This allows user to define a global max limit for max-keep-run value. If user has defined annotation on pipelineRun,

Reviewer: It would be great if you mention  to which resource max-keep-run value is specified

>   This allows user to define a max limit for max-keep-run value. If user has defined `max-keep-run` annotation on pipelineRun,

Reviewer 2: This let the user define a max limit for the max-keep-run value. When the user has defined a `max-keep-run` annotation on a pipelineRun   then its value should be less than or equal to the upper limit, otherwise upper limit will be used for cleanup.

>   then it's value should be less than or equal to max limit or max-keep-run value will be used from configmap.

Reviewer 2: same

* `default-max-keep-runs`

>   This allows user to define a default global limit for max-keep-run value. If defined then it applies to all pipelineRun
>   who doesn't have max-keep-runs annotation

Reviewer 3: ```suggestion
  This allows user to define a default global limit for max-keep-run value. If defined then it's applied to all the pipelineRun
  which do not have max-keep-runs annotation.
```
